src/Common/src/Interop/Windows/perfcounter/UnsafeNativeMethods.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+namespace Microsoft.Win32
+{
+    internal static class UnsafeNativeMethods
+    {
+        internal const string ADVAPI32 = "advapi32.dll";
+
+        internal const int ERROR_SUCCESS = 0x0;
```

Comment:
Remove these and use Interop.Errors directly

Reply:
fixed